### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by jcudmore**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,33 @@ All the data used in Thinkful's
 ## Contributing
 To update the data for your bootcamp, fork this repo and make a pull request.
 
+### Permissions
+We ask that only employees or those closely affiliated with a bootcamp update its data. Once a PR is submitted, Thinkful we verify the data and push the changes live. If you have any questions please contact bootcamps@thinkful.com.
```
Comment (rkuykendall) [inline README.md]: Thinkful **will** verify

File: README.md [hunk 2]
```diff
@@ -5,6 +5,33 @@ All the data used in Thinkful's
...
+
+*For now - we are only accepting bootcamp programs that teach programming and design.*
+
+### Updating your logo
+There is no file size or dimension limit, but a good logo will be large enough to look sharp on retina displays, while being small enough to allow the page to load quickly. Logos should be transparent PNG, ideally optimized using a tool such as pngcrush. A good target is 150px high, as that will allow it to be 2x the resolution of the current largest display.
+
+### Updating your school’s description
+- Descriptions should be no longer than 1000 characters and no links will be allowed in the text.
+- Copy should be written in the third person.
+- Descriptions should be as objective as possible. We will not accepted phrases like “one of the best bootcamps” but will accept “Rated a top 10 bootcamp by TNW”.
```
Comment (benjaminEwhite) [inline README.md]: "We will not accept"

File: README.md [hunk 3]
```diff
@@ -5,6 +5,33 @@ All the data used in Thinkful's
...
+
+### Updating individual stats
```
Comment (benjaminEwhite) [inline README.md]: This all would be better as a table

**Comment by rkuykendall** [comment]
After Ben's catch :+1:

**Comment by jcudmore** [comment]
Ok fixed "accepted". I will merge

**Pull request merged**